### ISSUE TITLE
Cards from shared Decks don't show up in the upcoming cards dashboard

### ISSUE THREAD

**Issue by varac** [nextcloud/deck#2353]
**Describe the bug**

Cards with a due-date set show up in the upcoming cards dashboard fine. 
However, cards with a due date set from shared decks won't show up in the upcoming cards dashboard fine.

**To Reproduce**

1. Set a due date on a card in a non-shared deck
2. Card will show up in the upcoming cards dashboard
3. Share the board (rw) with another NC user 
4. Card will disappear from the upcoming cards dashboard

**Expected behavior**

The card from the shared deck should show up in the upcoming cards dashboard

**Client details:**
 - OS: Ubuntu 20.04
 - Browser: Firefox
 - Version 81.0
 - Device: Desktop

**Server details**

Unfortunatly I don't have admin access to this hosted NC instance.

**Comment by juliushaertl** [comment]
Cards from shared boards will only show up if you are assigned to them. I would generally keep the behavior like this since otherwise the overview becomes quite cluttered with all cards with duedates being added there.

**Comment by juliushaertl** [comment]
cc @nextcloud/designers

**Comment by jakobroehrl** [comment]
Maybe we should show due cards of shared boards to all board users if there is no assignee.

**Comment by varac** [comment]
> Maybe we should show due cards of shared boards to all board users if there is no assignee.

That sounds reasonable, or make it configurable if unassigned/assigned issues show up in the upcoming cards dashboard.

**Comment by juliushaertl** [comment]
> Maybe we should show due cards of shared boards to all board users if there is no assignee.

This will be hard to detect for each board individually without having a major performance impact.

**Comment by Henschwen** [comment]
> 
> 
> > Maybe we should show due cards of shared boards to all board users if there is no assignee.
> 
> This will be hard to detect for each board individually without having a major performance impact.

I have a similar issue: I would like cards to show up in the "upcoming cards dashboard" that **are** assigned to users **via groups they are in**. 
Because I want to share some cards directly with a group/circle without having to assign everybody one by one.
Is this something that could be added rather quick without discussion, or would this also cause "a major performance impact"?

_PS: since I am new to Github, especially for this issue - shall I rather write a **new issue** only for this specific detail (the next time / still sould do so?)?_

thanks a lot! 
Hendrik

**Comment by prundukevich** [comment]
> 
> Maybe we should show due cards of shared boards to all board users if there is no assignee.

I would agree. There is a filter - so you can any time filter only your cards. I would like to discuss shared cards with my colleagues using talk on small standups, but there is no point where I can see all the cards same time. 
And even more they somehow disappear from view after the board is shared.

**Comment by CoolDreamZ** [comment]
@prundukevich where is the filter on the "Upcoming Cards" view? I can't find it

Perhaps we need an "All Cards" (visible to me) view?

**Comment by prundukevich** [comment]
My bad. It's really missing there. Then it's worth to implement it there as well..?

> @prundukevich where is the filter on the "Upcoming Cards" view? I can't find it
>
